src/main/java/org/isf/security/LoginApi.java

```diff
@@ -27,34 +27,35 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.ApiResponse;
-import io.swagger.annotations.ApiResponses;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
 /**
  * Here only for swagger
  *
  * @author antonio
  */
-@Api("Login")
+
 @RestController
+@Tag(name = "Login")
```

Comment:
Move `logout` endpoint to appear under the same `login` tag

```suggestion
@Tag(name = "Auth")
```